node/network/statement-distribution/src/tests.rs

```diff
@@ -1870,3 +1864,15 @@ fn make_session_info(validators: Vec<Pair>, groups: Vec<Vec<u32>>) -> SessionInf
 		random_seed: [0u8; 32],
 	}
 }
+
+fn get_metadata_assuming_seconded(
```

Comment:
nit: `get_..` -> `derive_*` or `create_*`  seems to be a more appropriate prefix